### PR TITLE
[doc] Revise web test suite developer guide

### PR DESCRIPTION
**Pull request by zqzhang**

### CODE REVIEW

**Comment by zqzhang** [comment]
@jennycao @Honry @Shao-Feng PTAL

You can check the visual at

https://github.com/zqzhang/crosswalk-test-suite/blob/dev-guide/doc/Web_Test_Suite_Developer_Guide.md

File: doc/Web_Test_Suite_Developer_Guide.md [hunk 1]
```diff
@@ -1,390 +1,308 @@
...
+  - `tct`: test suites leveraged from Tizen Compliance Tests.
+  - `webapi`: test suites checking compliance with Crosswalk Web API
+    specifications.
+  - `embedding`: test suites checking compliance with Crosswalk Embedding API
+    specifications.
+  - `wrt`: test suites checking compliance with web runtime core specification.
+  - `cordova`: test suites checking cordova-plugin-crosswalk-webview.
+  - `usecase`: test suites verifying Crosswalk features from end user's
+    perspective.
+  - `stability`: test suite checking stability.
```
Comment (Honry) [inline doc/Web_Test_Suite_Developer_Guide.md]: What about apptools?

File: doc/Web_Test_Suite_Developer_Guide.md [hunk 2]
```diff
@@ -1,390 +1,308 @@
...
+- `<module>`: abbreviation of Web API specification or function component.
+- `<category>`: origination of the API specifications or features:
+  - `w3c`: standard specifications from W3C.
+  - `css3`: W3C CSS3 specifications.
+  - `html5`: W3C HTML5 specification.
+  - `nonw3c`: supplementary API specifications from other standard orgnizations
+    or browser engines other than W3C.
+  - `wrt`: Crosswalk runtime core specification.
+  - `andriod`: Crosswalk Project for Andriod specific features.
```
Comment (Honry) [inline doc/Web_Test_Suite_Developer_Guide.md]: Can we add more platforms here? Such as linux, windows, and ios.

File: doc/Web_Test_Suite_Developer_Guide.md [hunk 3]
```diff
@@ -1,390 +1,308 @@
...
+  package's architecture in different package types. It will be parsed by the
+  pack tool (`pack.py`) when build web test suite package.
+- `tests.full.xml`, `tests.xml`: files to describe all test cases for this test
+  suite. See [Appendix 1](#appendix-1-testsfullxml-and-testsxml) for details.
+- `testcase.xsl`, `testresult.xsl`, `tests.css`: XSLT style for test case and
+  test result, used by `tests.full.xml`, `tests.xml`, etc.
+
+## Package and web application naming convention
+
+Tipically package name is in `<type>-<module>-<category>-tests` format.
```
Comment (Honry) [inline doc/Web_Test_Suite_Developer_Guide.md]: 's/Tipically/Typically/'

**Comment by zqzhang** [comment]
Thank you all for the review; I will address them in this week when handle other urgent things.

File: doc/Web_Test_Suite_Developer_Guide.md (hunk 1 above)
Comment (zqzhang) [inline doc/Web_Test_Suite_Developer_Guide.md]: Feel free to add it after this pull request is merged, thanks.

File: doc/Web_Test_Suite_Developer_Guide.md (hunk 2 above)
Comment (zqzhang) [inline doc/Web_Test_Suite_Developer_Guide.md]: Feel free to add them after this pull request is merged, thanks.

File: doc/Web_Test_Suite_Developer_Guide.md (hunk 3 above)
Comment (zqzhang) [inline doc/Web_Test_Suite_Developer_Guide.md]: Good catch, thanks.

**Comment by zqzhang** [comment]
@jennycao I don't know where your last 2 comments target for.

**Comment by zqzhang** [comment]
@Honry @jennycao your comments shall have been addressed; could you please check https://github.com/zqzhang/crosswalk-test-suite/commit/30fc17e7f90600fa08aa4924f98c416374d4f177? Thanks.

**Pull request merged**